tests/utils.go

```diff
@@ -884,7 +884,12 @@ func DeleteRawManifest(object unstructured.Unstructured) error {
 
 	uri := composeResourceURI(object)
 	uri = uri + "/" + object.GetName()
-	result := virtCli.CoreV1().RESTClient().Delete().RequestURI(uri).Do()
+
+	options := &metav1.DeleteOptions{}
```

Comment:
just reorder a little
```
policy := metav1.DeletePropagationBackground
options := &metav1.DeleteOptions{PropagationPolicy: &policy}
```

Reply:
Thanks for reviewing! PR updated accordingly.